### PR TITLE
Add solar schedules

### PR DESCRIPTION
**Pull request by mheppner**
This is in response for #8. I'm not sure what you want the names to be ("solars" for `verbose_name_plural` seems odd, for example). It uses the event names directly from `celery.schedules.solar`, so the model doesn't have any constants. Is it safe to redefine them here?

### CODE REVIEW

**Comment by mheppner** [comment]
Migrations fail because the `label` set in `django_celery_beat.apps.BeatConfig` should be `django_celery_beat`, not `beat`.

**Comment by auvipy** [comment]
rebase and try to fix the errors

**Comment by mheppner** [comment]
@auvipy ok this is cleaned up now

File: django_celery_beat/models.py [hunk 1]
```diff
@@ -28,13 +28,61 @@
...
+    )
+    longitude = models.DecimalField(
+        _('latitude'), max_digits=9, decimal_places=6
+    )
+
+    class Meta:
+        """Table information."""
+
+        verbose_name = _('solar')
+        verbose_name_plural = _('solars')
```
Comment (ask) [inline django_celery_beat/models.py]: maybe this should be 'solar event' and 'solar events'?

Comment (mheppner) [inline django_celery_beat/models.py]: sounds good to me, thanks ask :)

Comment (mheppner) [inline django_celery_beat/models.py]: sorry @ask, i forgot to check travis last night. the `unique_together` has been added.

**Pull request merged**